src/pages/MFARequiredScreen.vue

```diff
@@ -24,56 +24,63 @@ let mfaWindow: Window | null
 const storage = getStorage()
 
 async function handleProceed() {
-  cleanExit = false
-  const mfaSetupPath = new URL(`mfa/${appStore.id}/setup`, AUTH_URL)
-  if (appStore.standaloneMode == 0) {
-    mfaWindow = window.open(
-      mfaSetupPath.toString(),
-      '_blank',
-      getWindowFeatures()
-    )
-    const handler = async (event: MessageEvent) => {
-      if (!event?.data?.status) {
-        return
-      }
-      cleanExit = true
-      const data = event.data
+  const { loginType } = JSON.parse(
+    storage.session.getItem('userInfo') as string
+  )
+  if (loginType === 'firebase') {
```

Comment:
It's better to use a Set membership check here, we are more likely to add more bearer type authentication methods.

Reply:
created a util that checks this based on a list [d256a90](https://github.com/arcana-network/wallet-ui/pull/353/commits/d256a909bd1547d303682852f8bf01298c7ad0ba)